pkgs/top-level/all-packages.nix

```diff
@@ -257,8 +257,8 @@ with pkgs;
     fetcherArgs = (if fetchSubmodules
         then { inherit rev fetchSubmodules; url = "${baseUrl}.git"; }
         else ({ url = "${baseUrl}/archive/${rev}.tar.gz"; } // privateAttrs)
-      ) // passthruAttrs // { inherit name; };
-  in fetcher fetcherArgs // { meta.homepage = baseUrl; inherit rev; };
```

Comment:
Actually, wouldn't it work to just use `inherit owner repo rev;` here?

Reply:
I'll test this. Then probably not a single derivation will even change, because just overriding the derivation attribute sets attributes doesn't influence the drv.

Reply:
Turns out neither of those approaches influence the derivation, so I think `passthru` is the cleaner way to do this then. `passthru` is exactly meant for this kind of stuff.